packages/cozy-client/src/withMutate.jsx

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react'
+
+const withMutate = (mutation, options = {}) => WrappedComponent => {
+  return class Wrapper extends Component {
```

Comment:
We shall name the wrapper component with a more specific name for debugging with react devtools like `WithMutate` for instance.

Reply:
Or you can play with the name after its instantiation: see https://reactjs.org/docs/higher-order-components.html#convention-wrap-the-display-name-for-easy-debugging